src/components/frontend-engine/types.ts

```diff
@@ -49,8 +50,10 @@ export type TRevalidationMode = Exclude<keyof ValidationMode, "onTouched" | "all
 export type TValidationMode = keyof ValidationMode;
 
 export interface IFrontendEngineRef extends HTMLFormElement {
-	/** gets information about the entire form state */
-	getFormState: () => FormState<TFrontendEngineValues>;
+	/** gets form values */
+	getValues: () => TFrontendEngineValues;
+	/** check if form is valid */
+	isValid: () => boolean;
```

Comment:
should this be optional as well?

Reply:
not really because they will always be defined in `useImperativeHandle`